maquetado/index.html

```diff
@@ -11,21 +11,28 @@
 </head>
 
 <body>
-  <div class="book-detail">
-    <p class="book-title">Titulo del Libro</p>
-    <p class="book-description book-genre">(género)</p>
-  </div>
-  <div class="book-detail">
-    <p class="book-titleDescription">Autor del libro:</p>
-    <p class="book-description">Nombre del autor del libro</p>
-  </div>
-  <div class="book-detail">
-    <h1 class="book-titleDescription">Editorial:</h1>
-    <p class="book-description">Nombre de la editorial</p>
-  </div>
-  <div class="book-detail">
-    <p class="book-titleDescription">Año de publicación:</p>
-    <p class="book-description">Año de publicación</p>
+  <div class="book-content">
+    <img class="image" src="image/book-cover.png" alt="Book Image">
```

Comment:
Change the class name `image` for a more descriptive name.

Reply:
Agree. You could use `book-cover`. Try to think of it as if you could have more than one different type of image, so the best it's to use components like classes for this. 

The alt attribute could also be `Book cover`